Edge/edge on kubernetes (#1399)
SoftwareAG/c8y-docs: content/edge-k8s/edge-custom-resource-definition-bundle/k8-edge-mongodb-storage-size.md

---
weight: 40
title: MongoDB storage size
layout: redirect
---

|<div style="width:150px">Field</div>|Required|<div style="width:70px">Type</div>|Default|Description|
|:---|:---|:---|:---|:---|

> |storage|No|string|Default to 75 GB|Specify the PVC storage.<br>{{< c8y-admon-info >}}Once {{< product-c8y-iot >}} Edge is installed, you can only increase this value and cannot reduce.{{< /c8y-admon-info >}}

Reviewer: ```suggestion
|storage|No|string|Default to 75 GB|Specify the PVC storage.<br>Once {{< product-c8y-iot >}} Edge is installed, you can only increase this value, but not reduce.
```

Author: This is intended.

Reviewer: there's a ton of admonitions in the tables and it's a case of double markup (table and admonition) which makes the content really crowded.
if you insist on keeping all of them, go ahead. it's just generally something we would not recommend.

Author: We need some sort of identification for that text. I'll just use **Note:**. Hope that is fine?
